vector-im/element-android
# [Location sharing] Align naming of components for live location feature

## Type of change

- [x] Other : Renaming

## Content


Align the naming of all components involved into the live location sharing feature.

## Motivation and context


For consistency purpose. Closes #6647 

## Screenshots / GIFs





## Tests



- Check the app build
- Start a live into a room
- Check it is correctly started
- Stop the live
- Check it is correctly stopped

## Tested devices

- OS version(s): Android 11

## Checklist
